packages/sign_in_with_apple/lib/sign_in_with_apple.dart

```diff
@@ -44,4 +46,8 @@ class SignInWithApple {
       ),
     );
   }
+
+  static Future<bool> isAvailable() {
+    return channel.invokeMethod<bool>('isAvailable');
```

Comment:
```suggestion
    return _isAvailable ??= channel.invokeMethod<bool>('isAvailable');
```

So we cache this for cases where the button is not always displayed?

Then the signature could even become `FutureOr<bool>` and we an immediate value in most cases.

Reply:
@HenriBeck Ah, it's cached on the widget (but then still wrapped inside a Future). Does `FutureBuilder` have some private access to directly render with a resolved future? Didn't see any public API for that so I would be surprised…